sample.ini

```diff
@@ -1,29 +1,37 @@
 [General]
-
-hostname = 0.0.0.0
 
-portNumber = 8000
+hostname = 0.0.0.0
 
-enableTracing = 0
+portNumber = 8000
+
+enableTracing = 0
 
 ; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
-
+
 ; remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
-
-
-; These options are unlikely to need editing by users
 
-maxCacheEntries = 2000000
 
-mruSymbolStateFile = /tmp/snappy-mru-symbols.json
+; These options are unlikely to need editing by users
+
+maxCacheEntries = 2000000
+
+mruSymbolStateFile = /var/snappy-mru-symbols.json
 maxMRUSymbolsPersist = 10
 
-[SymbolPaths]
+[Log]
+
+maxFiles = 10
+
+maxFileSize = 1048576
+
+logPath = /tmp/log/snappy
+
+[SymbolPaths]
 
-Firefox = /mnt/netapp/breakpad/symbols_ffx
+Firefox = /mnt/netapp/breakpad/symbols_ffx
```

Comment:
This one I could not find a reasonable solution. If you comment out the local paths, them snappy will not try any local search. If you comment out the symbol URL, and remove [this line](https://github.com/mozilla/Snappy-Symbolication-Server/blob/master/symbolicationWebService.py#L46), no S3 fetch happens.